Infrastructure-Scripts/Remove-AzSqlIpException.ps1

```diff
@@ -0,0 +1,58 @@
+<#
+    .SYNOPSIS
+    Remove the whitelisted IP from the SQL Server firewall rule
+
+    .DESCRIPTION
+    Remove the whitelisted IP from the SQL firewall rule
+
+    .PARAMETER Name
+    The name of the firewall rule
+
+    .PARAMETER IPAddress
+    The whitelisted IP Address that will get removed by this script
+
+    .PARAMETER  ResourceNamePattern
+    Substring of the SQL Server to search for
+
+    .EXAMPLE
+    Remove-AzureSQLIPException -Name rulename -IPAddress $(IPAddress) -ResourceNamePatterb das-*
+#>
+[CmdletBinding()]
+param(
+    [ValidateNotNull()]
+    [IPAddress]$IPAddress,
+    [ValidateNotNull()]
+    [Parameter(Mandatory = $true)]
+    [string]$ResourceNamePattern,
+    [ValidateNotNull()]
+    [Parameter(Mandatory = $true)]
+    [string]$Name
+)
+
+$SubscriptionSqlServers = Get-AzResource -Name $ResourceNamePattern -ResourceType "Microsoft.Sql/Servers"
+
+foreach($SqlServer in $SubscriptionSqlServers) {
+    $ResourceGroupName = $SqlServer.ResourceGroupName
+    $ServerName = $SqlServer.Name
+}
```

Comment:
The flow for this foreach wouldn't loop through the servers and remove the whitelist from all of them

Reply:
May also want a unit test scenario where Get-AzResource returns more than 1 SqlServer